FrontEnd/src/Appointment/AppointmentPage.css

```
@@ -17,9 +17,11 @@
   padding: 0 1rem;
 }
 
-h2,
-p {
-  color: var(--darkTwo);
+h2{
```

Seems like lots of improperly formatted css and occasionally JS. Don't forget to apply the formatter as the last step before committing your code.

```suggestion
h2 {
```

---

Just formatted all the files! Thanks